test/add-member.test.js

```diff
@@ -43,10 +43,15 @@ test('get added to a group', async (t) => {
   } = await alice.tribes2.create().catch(t.fail)
   t.pass('alice created a group')
 
-  await alice.tribes2.addMembers(groupId, [bobRoot.id])
+  console.log({ groupId, bobRootId: bobRoot.id })
+  await alice.tribes2.addMembers(groupId, [bobRoot.id]).catch(t.fail)
+  console.log('added bob')
   t.pass('alice added bob to the group')
 
-  await replicate(alice, bob, { waitUntilMembersOf: groupId })
+  console.log('about to replicate')
+  await replicate(alice, bob).catch(t.fail)
```

Comment:
it gets stuck here, idk why. the only real difference up to this point should be that start() doesn't automatically add invites, so bob doesn't automatically start decrypting alice's group feeds. but bob should still replicate it since i've added `person2.ebt.request(feedId, true)`, right?

Reply:
had a call with andre and we figured out that the problem was that both users had the same feeds fully replicated, the problem was that we were comparing trees too and not only feeds. bob had alice's group feed but he didn't know it was alice's because he couldn't yet decrypt the announce message. so we just removed all the tree equality code. hopefully we don't actually need that?